app/src/main/res/layout/activity_main.xml

```diff
@@ -156,92 +155,117 @@
+                            android:paddingEnd="10dp"
+                            android:text="@string/image_name" />
+                    </FrameLayout>
 
-                            <androidx.appcompat.widget.SwitchCompat
-                                android:id="@+id/main_switch_resize_image_left"
-                                android:layout_width="wrap_content"
-                                android:layout_height="48dp"
-                                android:layout_gravity="bottom|center_horizontal"
-                                android:text="@string/resize"
-                                android:textColor="#EEEEEE"
-                                android:textSize="18sp"
-                                tools:ignore="DuplicateSpeakableTextCheck" />
+                </TableRow>
 
-                        </FrameLayout>
+                <TableRow
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent">
 
-                        <FrameLayout
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
+                    <LinearLayout
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:paddingTop="20dp"
+                        android:paddingBottom="20dp"
+                        android:paddingStart="12dp"
+                        android:paddingEnd="12dp"
+                        android:orientation="horizontal">
+
+                        <Space
+                            android:layout_width="0dp"
+                            android:layout_height="1dp"
                             android:layout_weight="1">
+                        </Space>
 
-                            <ImageButton
-                                android:id="@+id/home_button_rotate_image_left"
-                                android:layout_width="48dp"
-                                android:layout_height="48dp"
-                                android:layout_gravity="bottom|center_horizontal"
-                                android:backgroundTint="#F57C00"
-                                android:contentDescription="@string/rotate_image_left"
-                                app:srcCompat="@android:drawable/stat_notify_sync_noanim"
-                                tools:ignore="ImageContrastCheck" />
-                        </FrameLayout>
+                        <androidx.appcompat.widget.SwitchCompat
+                            android:id="@+id/main_switch_resize_image_left"
+                            android:layout_width="wrap_content"
+                            android:layout_height="48dp"
+                            android:gravity="center"
+                            android:text="@string/resize"
+                            tools:ignore="DuplicateSpeakableTextCheck" />
```

Comment:
Can you please re-add these properties to both Resize SwitchCompats in activity_main.xml 

```
android:textColor="#EEEEEE"
android:textSize="18sp"
```